gen-ai/orchestrator-server/src/main/python/server/tests/services/test_rag_chain.py

```diff
@@ -100,25 +100,26 @@ async def test_rag_chain(
         },
         'question_answering_prompt_inputs': {
             'question': 'How to get started playing guitar ?',
-            'no_answer': "Sorry, I don't know.",
+            'no_answer': 'Sorry, I don t know.',
             'locale': 'French',
         },
         'embedding_question_em_setting': {
             'provider': 'OpenAI',
             'api_key': {'type': 'Raw', 'value': 'ab7***************************A1IV4B'},
             'model': 'text-embedding-ada-002',
         },
-        'document_index_name': 'my-index-name',
-        'document_search_params': {
+        'vector_store_setting': {
             'provider': 'OpenSearch',
-            'filter': [
-                {
-                    'term': {
```

Comment:
Filter are still available in document_search_param.

Reply:
yes it's a mistake

Reply:
I see that it's still removed in the change, can you add it back ?